cmd/cql/internal/permission.go

```diff
@@ -0,0 +1,93 @@
+package internal
+
+import (
+	"encoding/json"
+
+	"github.com/CovenantSQL/CovenantSQL/client"
+	"github.com/CovenantSQL/CovenantSQL/proto"
+	"github.com/CovenantSQL/CovenantSQL/types"
+)
+
+var CmdPermission = &Command{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var CmdPermission should have comment or be unexported